resources/js/processes-catalogue/components/DefaultTab.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="d-flex align-items-center justify-content-center flex-column">
+    <img
+      :src="imageSrc"
+      :alt="altText"
+    />
+    <div class="text-center">
+      <strong>{{ titleText }}</strong>
+    </div>
+    <div class="text-center">
+      {{ descriptionText }}
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    imageSrc: {
+      type: String,
+      default: "/img/launchpad-images/frame.png",
```

Comment:
please rehuse the default: "/img/tasks-loading.svg",

Reply:
solved